example/neubot/client_dash.cpp

```diff
@@ -0,0 +1,54 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "src/common/utils.hpp"
+#include <functional>
+#include <iostream>
+#include <measurement_kit/common.hpp>
+#include <measurement_kit/ext.hpp>
+#include <measurement_kit/http.hpp>
+#include <measurement_kit/neubot.hpp>
+#include <stdlib.h>
+#include <string>
+#include <unistd.h>
+
+using namespace mk;
+using namespace mk::neubot::negotiate;
+using namespace mk::net;
+using namespace mk::http;
+
+static const char *kv_usage =
+    "usage: ./example/http/client_dash [-vn] [-a address]\n";
+
+int main(int argc, char **argv) {
+    Settings settings;
+    settings["http/url"] = "http://127.0.0.1";
```

Comment:
It would be optimal if this key was called `url` and the test machinery provides to set up `http/url` according to the value of `url`. It would also be optimal if, when the `url` is empty, it is looked up using mlabns.

If the latter is too complex to implement before the end of the internship, never mind. But, to start off, I'd like to see the address of a real mlab server to be used as default (e.g. `neubot.mlab.mlab1.mil01.measurement-lab.org`).